build-aux/logfile-uploader.py

```diff
@@ -41,6 +42,10 @@ def collect_logfiles():
     # JS harness
     for log in glob.glob('dev/logs/node1.log'):
         tb.add(log)
+    # couchjs OS process IO logs
+    if os.path.isdir(OSPROC_IO_LOG_DIR):
```

Comment:
No need for this check, if the directory doesn't exist, `glob.glob` will return `[]`:

```bash
$ ls -la /tmp/abc/def
ls: cannot access /tmp/abc/def: No such file or directory
$ python
Python 2.7.9 (default, Jun 29 2016, 13:08:31)
[GCC 4.9.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> import glob
>>> glob.glob("/tmp/abc/def/*")
[]
```